messages/es.json

```
@@ -164,13 +164,13 @@
   "promotions.promotion.restriction.restrictTradePolicies.verb.not.any": "no es una de",
   "promotions.promotion.restriction.restrictTradePolicies.placeholder": "Seleccione una política comercial",
   "promotions.promotion.restriction.origin": "Origen",
-  "promotions.promotion.restriction.origin.marketplace": "Marketplace",
-  "promotions.promotion.restriction.origin.fulfillment": "Fulfillment",
+  "promotions.promotion.restriction.origin.marketplace": "Mercado",
```

These two are also known terms
```suggestion
  "promotions.promotion.restriction.origin.marketplace": "Marketplace",
```